components/provisioner/internal/installation/download/components/tar.go

```diff
@@ -0,0 +1,29 @@
+package components
+
+import (
+	"os"
+
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/archive"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/download"
+
+	"github.com/pkg/errors"
+)
+
+func DownloadTgz(URL string, tmpDst string) error {
+	path, err := download.GetFile(URL, tmpDst)
+	if err != nil {
+		return errors.Wrap(err, "while downloading tgz archive")
+	}
+
+	err = archive.Untar(path, tmpDst)
+	if err != nil {
+		return errors.Wrap(err, "while unzipping tgz archive")
```

Comment:
Logic does not support "file://" protocols.

Reply:
After discussion, it looks like an extra feature for future implementation.